watertap/examples/flowsheets/case_studies/full_water_resource_recovery_facility/BSM2.py

```diff
@@ -371,6 +365,10 @@ def set_operating_conditions(m):
     m.fs.RADM.volume_vapor.fix(300)
     m.fs.RADM.liquid_outlet.temperature.fix(308.15)
 
+    # Dewatering Unit - fix either HRT or volume.
+
+    # Set specific energy consumption averaged for centrifuge
+    m.fs.unit.energy_electric_flow_vol_inlet[0] = 0.069 * pyunits.kWh/pyunits.m**3
```

Comment:
```suggestion
    m.fs.DU.energy_electric_flow_vol_inlet[0] = 0.069 * pyo.units.kWh/pyo.units.m**3
```
I'm assuming this is what you meant @adam-a-a?

Reply:
Yes- thanks for the catch.